### PR TITLE
stats8 pr moved here

### PR DESCRIPTION
**Pull request by dogusmiuw**

### CODE REVIEW

**Review by dogusmiuw — COMMENTED** [review]
code approved. waiting for ui review

File: editor-components/stats/stats8/stats8.module.scss
Comment (Ayazelar) [inline editor-components/stats/stats8/stats8.module.scss:1]: you can use variables in scss. please check

File: editor-components/stats/stats8/stats8.tsx [hunk 1]
```diff
@@ -3,112 +3,300 @@ import { BaseStats } from "../../EditorComponent";
...
+              <div className={this.decorateCSS("content")}>
+                {title && (
+                  <h2
+                    className={
+                      this.decorateCSS("title") +
+                      " " +
+                      this.decorateCSS("text-uppercase")
+                    }
+                  >
+                    {title}
```
Comment (Ayazelar) [inline editor-components/stats/stats8/stats8.tsx:210]: use this.getPropValue cause you are using cast to string above. if you are using string the inline editor does not appear in strings

**Review by Ayazelar — CHANGES_REQUESTED** [review]
.

**Review by Ayazelar — APPROVED** [review]

**Pull request closed without merging**